docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1825,12 +1838,50 @@ function updateMultistreamVideoLayout() {
   }
 }
 
+async function getStatsForVideoPane(meeting, videoPane) {
+  const {remoteMedia} = videoPane;
```

Comment:
are there plans to have this function inside the SDK or WCF ?

Reply:
yeah, I think longer term we should have WCME API to get the stats for a specific receive slot and then we can expose that API through SDK/WCF. Right now it's just a hack and as I'm accessing private property I can only do it from js, not typescript so I've put it in the sample app